### PR TITLE
[MINOR][DOCS] Fix few typos in structured streaming doc

### PR DESCRIPTION
**Pull request by ramkumarvenkat**
## What changes were proposed in this pull request?

Minor typo in `even-time`, which is changed to `event-time` and a couple of grammatical errors fix.

## How was this patch tested?

N/A - since this is a doc fix. I did a jekyll build locally though.

### CODE REVIEW

**Comment by ramkumarvenkat** [comment]
@srowen @tdas Can you guys please look into this small doc fix?

File: docs/structured-streaming-programming-guide.md [hunk 1]
```diff
@@ -392,7 +392,7 @@ data, thus relieving the users from reasoning about it. As an example, let’s
 see how this model handles event-time based processing and late arriving data.
 
 ## Handling Event-time and Late Data
-Event-time is the time embedded in the data itself. For many applications, you may want to operate on this event-time. For example, if you want to get the number of events generated by IoT devices every minute, then you probably want to use the time when the data was generated (that is, event-time in the data), rather than the time Spark receives them. This event-time is very naturally expressed in this model -- each event from the devices is a row in the table, and event-time is a column value in the row. This allows window-based aggregations (e.g. number of events every minute) to be just a special type of grouping and aggregation on the even-time column -- each time window is a group and each row can belong to multiple windows/groups. Therefore, such event-time-window-based aggregation queries can be defined consistently on both a static dataset (e.g. from collected device events logs) as well as on a data stream, making the life of the user much easier.
```
Comment (tdas) [inline docs/structured-streaming-programming-guide.md]: can you point out what changed here? github doesnt seeming to showing the difference clearly like the other diffs.

Comment (ramkumarvenkat) [inline docs/structured-streaming-programming-guide.md]: and aggregation on the `even-time column`

is changed to 

and aggregation on the `event-time column`

File: docs/structured-streaming-programming-guide.md [hunk 2]
```diff
@@ -647,7 +647,7 @@ df.groupBy("deviceType").count()
 </div>
 
 ### Window Operations on Event Time
-Aggregations over a sliding event-time window are straightforward with Structured Streaming. The key idea to understand about window-based aggregations are very similar to grouped aggregations. In a grouped aggregation, aggregate values (e.g. counts) are maintained for each unique value in the user-specified grouping column. In case of window-based aggregations, aggregate values are maintained for each window the event-time of a row falls into. Let's understand this with an illustration. 
+Aggregations over a sliding event-time window are straightforward with Structured Streaming. The key idea to understand window-based aggregations is very similar to grouped aggregations. In a grouped aggregation, aggregate values (e.g. counts) are maintained for each unique value in the user-specified grouping column. In case of window-based aggregations, aggregate values are maintained for each window the event-time of a row falls into. Let's understand this with an illustration.
```
Comment (srowen) [inline docs/structured-streaming-programming-guide.md]: This still needs a fix -- I would just say "Window-based aggregations are very similar to grouped aggregations"

**Review by srowen — COMMENTED** [review]
We'd generally ask you scan the rest of the docs and similar docs for typos too, in order to cut down on the number of typo fix PRs we need to review.

File: docs/structured-streaming-programming-guide.md (hunk 2 above)
Comment (tdas) [inline docs/structured-streaming-programming-guide.md]: agreed. 
"The key idea to understand is that window-based aggregations are very similar to grouped aggregations."

Comment (ramkumarvenkat) [inline docs/structured-streaming-programming-guide.md]: Fixed this as well

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**